src/test/java/Homework17CH.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17CH extends BaseTest {
+
+    @Test
+    public static void AddSongToPlaylist() throws InterruptedException {
+
+        navigateToPage();
+
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        allSongsPage();
+        firstSong();
+        Thread.sleep(2000);
+        addSongTo();
+        Thread.sleep(2000);
+        firstPlaylist();
+        Thread.sleep(2000);
+        validatePlaylist();
+        Thread.sleep(2000);
+
```

Comment:
It's a bad practice to use Thread.sleep()  - Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.